besu/src/main/java/org/hyperledger/besu/cli/options/TransactionPoolOptions.java

```diff
@@ -247,15 +248,16 @@ static class Sequenced {
   }
 
   @CommandLine.ArgGroup(validate = false)
-  private final TransactionPoolOptions.Unstable unstableOptions =
-      new TransactionPoolOptions.Unstable();
+  private final Unstable unstableOptions = new Unstable();
 
   static class Unstable {
     private static final String TX_MESSAGE_KEEP_ALIVE_SEC_FLAG =
         "--Xincoming-tx-messages-keep-alive-seconds";
-
     private static final String ETH65_TX_ANNOUNCED_BUFFERING_PERIOD_FLAG =
         "--Xeth65-tx-announced-buffering-period-milliseconds";
+    private static final String MAX_TRACKED_SEEN_TXS_PER_PEER = "--Xmax-tracked-seen-txs-per-peer";
```

Comment:
nit: not sure there's a convention for this but maybe could start "--Xpeer-..."` similar to other groups of config.

Reply:
Added the fallback value